homepage.php

```diff
@@ -49,57 +49,57 @@
 		<div class="content_4">
 			<Space></Space>
 			<div class="images1">
-				<a href = "Query8.php"><img src = "images/indie.jpg" class="image1"></a>
+				<a href = "Query8.php"><img src = "images/Indie2.jpg" class="image1"></a>
```

Comment:
30/8/2021: I trailing for some img for it is more match so it could use as an indicator to show the user that the img is the link to a query that contain all the song that is like the img represent. Also decided a nice img so it can improve the aesthetic of the img.